test/controllers/eventController.spec.js

```diff
@@ -0,0 +1,420 @@
+let mocha = require('mocha')
```

Comment:
ya I wasn't sure if needed to create new users to meet the diff requirements. I think I do but I wasn't sure where to put that